browser/src/Services/Learning/Tutorial/Tutorials/SearchInBufferTutorial.tsx

```diff
@@ -0,0 +1,81 @@
+/**
+ * TutorialManager
+ */
+
+import * as React from "react"
+
+import { ITutorial, ITutorialMetadata, ITutorialStage } from "./../ITutorial"
+// import { InitializeBufferStage, MoveToGoalStage } from "./../Stages"
```

Comment:
I think we can just remove this line if it's not used.

Reply:
Shoot, another case where I think @muream inherited my mistake (I see this in `BasicMovementTutorial.tsx` too).